roles/kubespray-defaults/vars/main.yml

```diff
@@ -27,3 +27,8 @@ kube_pods_subnets: >-
   {%- else -%}
   {{ kube_pods_subnet_ipv6 }}
   {%- endif -%}
+
+# Symlinks to etcd certs
+kube_etcd_cacert_file: "kube-client-ca.pem"
+kube_etcd_cert_file: "kube-client-cert.pem"
+kube_etcd_key_file: "kube-client-key.pem"
```

Comment:
Seems to conflict with cilium / calico (with etcd datatstore presumably), given the test results.

Reply:
yes, I'll have to dig a bit more here. maybe need to run the symlink tasks from the calico/cilium roles